Content.Client/_NF/ShuttleRecords/UI/ShuttleRecordsWindow.xaml

```diff
@@ -0,0 +1,70 @@
+﻿<controls:FancyWindow xmlns="https://spacestation14.io"
+                      xmlns:graphics="clr-namespace:Robust.Client.Graphics;assembly=Robust.Client"
+                      xmlns:controls="clr-namespace:Content.Client.UserInterface.Controls"
+                      Title="{Loc 'shuttle-records-console-window-title'}"
+                      MinSize="700 300">
+
+    <BoxContainer Orientation="Vertical">
+
+        <PanelContainer>
+            <PanelContainer.PanelOverride>
+                <graphics:StyleBoxFlat BackgroundColor="#1B1E1B" />
+            </PanelContainer.PanelOverride>
+
+            <GridContainer Columns="3" HorizontalExpand="True" Margin="10">
+                <Label Text="{Loc 'id-card-console-window-target-id'}" />
+                <Button Name="TargetIdButton" Access="Public" />
+                <Label Name="TargetIdLabel" />
+                <Label Text="{Loc 'id-card-console-window-ship-name-label'}" />
+                <Label Name="ShipNameLabel" />
+            </GridContainer>
+        </PanelContainer>
+
+        <BoxContainer HorizontalExpand="True" VerticalExpand="True" Orientation="Horizontal" Margin="10">
+
+            <BoxContainer Orientation="Vertical" Margin="0 0 10 0">
+                <BoxContainer Orientation="Vertical" Margin="0 0 0 5">
+                    <Label Text="{Loc 'shuttle-records-search'}" StyleClasses="LabelSubText" Margin="0 0 5 0" />
+                    <BoxContainer Orientation="Horizontal">
+                        <LineEdit Name="SearchText" Access="Public" MinHeight="30" SetWidth="200" />
+                        <Button Name="ClearSearchButton" Access="Public" Text="✖" Margin="5 0 0 0" />
+                    </BoxContainer>
+                    <CheckBox Name="ShowActiveOnlyCheckbox" Text="{Loc 'shuttle-records-show-active-only'}" />
+                </BoxContainer>
+
+                <BoxContainer Orientation="Vertical" MinSize="250 300">
```

Comment:
If you add two VerticalExpand values (everything cascading down to the shuttle list), you end up getting additional shuttle records when expanding the window (good, IMO)
```suggestion
            <BoxContainer Orientation="Vertical" Margin="0 0 10 0" VerticalExpand="True">
                <BoxContainer Orientation="Vertical" Margin="0 0 0 5">
                    <Label Text="{Loc 'shuttle-records-search'}" StyleClasses="LabelSubText" Margin="0 0 5 0" />
                    <BoxContainer Orientation="Horizontal">
                        <LineEdit Name="SearchText" Access="Public" MinHeight="30" SetWidth="200" />
                        <Button Name="ClearSearchButton" Access="Public" Text="✖" Margin="5 0 0 0" />
                    </BoxContainer>
                    <CheckBox Name="ShowActiveOnlyCheckbox" Text="{Loc 'shuttle-records-show-active-only'}" />
                </BoxContainer>

                <BoxContainer Orientation="Vertical" MinSize="250 300" VerticalExpand="True">
```

Output at default size (100% scaling):
![image](https://github.com/user-attachments/assets/4709db23-531c-4910-b6ae-11e1a607b75a)

Extra records when grown:
![image](https://github.com/user-attachments/assets/d23c4efe-9101-4c87-a47e-c2c6a3b81fa8)